### ISSUE TITLE
Serious bug in DCLP saving!!

### ISSUE THREAD

**Issue by jcowey** [papyri/sosol#285]
Based on observations made in #284,

I think that saving in DCLP is not possible. Is there a bug that has crept in? Seems to have occurred yesterday.

**Comment by ryanfb** [comment]
It looks like XML validation for all DCLP is now throwing the error "White spaces are required between publicId and systemId.". There haven't been any deploys in over a week so nothing has changed in the codebase. There was an internet-wide routing error yesterday morning, which may have caused a networking issue in something loaded by the validation process, so as a first pass I'm going to try restarting the editor and seeing if that fixes it.

**Comment by hcayless** [comment]
Just as a guess, I think that error message is the result of trying to parse an error response in HTML5 (with a `<!DOCTYPE html>`). So it could be a routing error or similar.

**Comment by ryanfb** [comment]
Saving DCLP works fine for me after cycling the editor, and I don't see any errors logged since then. @jcowey does all look ok now to you?

**Comment by ryanfb** [comment]
@hcayless is right. The RNG URI used for DCLP is `http://www.stoa.org/epidoc/schema/8.23/tei-epidoc.rng` which now returns an HTML document with:

```
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>302 Found</title>
</head><body>
<h1>Found</h1>
<p>The document has moved <a href="https://www.stoa.org/epidoc/schema/8.23/tei-epidoc.rng">here</a>.</p>
</body></html>
```

I'll check the other RNG URIs for similar problems and roll new commit + deploy that should fix this.

**Comment by ryanfb** [comment]
Closing as this should now be actually fixed on production by 2a12ace. Please reopen if there are still problems.
